book/src/08_parallel_join.md

```diff
@@ -2,11 +2,13 @@
 
 As mentioned in the chapter dedicated on how to [dispatch][c3] systems,
 Specs automatically parallelizes system execution when there are non-conflicting
-system data requirements (every system data may only be accessed
-mutably once xor immutably from multiple systems).
+system data requirements (Two `System`s conflict if their `SystemData` needs access
+to the same resource where at least one of them needs write access to it).
 
 [c3]: ./03_dispatcher.html
 
+## Basic parallization
```

Comment:
Typo in parallelization.

Reply:
I constantly make that typo, thanks for spotting!